fuzzers/forkserver_simple/src/main.rs

```diff
@@ -51,6 +49,11 @@ pub fn main() {
                 .long("timeout")
                 .default_value("1200"),
         )
+        .arg(
+            Arg::new("arguments")
```

Comment:
I think clap can somehow collect everything beind a `--` as additional params, this could be cooler here

Reply:
https://docs.rs/clap/3.0.0-beta.5/clap/enum.AppSettings.html#variant.TrailingVarArg

That's the option I think

Reply:
it already does so. with `.setting(clap::ArgSettings::MultipleValues)`